lib/web/css/source/lib/_resets.less

```diff
@@ -105,6 +105,13 @@
             .lib-css(box-shadow, @focus__box-shadow);
         }
     }
+
+    input[type="radio"],
+    input[type="checkbox"] {
+        &:focus { 
+            .lib-css(box-shadow, none);
```

Comment:
Hi! 
1) .lib-css(); wrapping is not needed as it has no variables inside to check
2) I would just change line 101 to instead  (just my opinion) 
`input:not([disabled]):not([type="radio"]):not([type="checkbox"])`
Cheers